.github/workflows/lint.yaml

```
@@ -15,15 +15,16 @@ jobs:
         with:
           python-version: 3.8
 
-      - name: Install dependencies
+      - name: Install dependencies using poetry
         run: |
-          pip install -r dev-requirements.txt
+          pip install poetry
+          poetry install
 
       # We will not run `flake8` here, as we will use a separate flake8
       # action. As pre-commit does not support user installs, we set
       # PIP_USER=0 to not do a user install.
       - name: Run pre-commit hooks
-        run: export PIP_USER=0; SKIP=flake8 pre-commit run --all-files
+        run: export PIP_USER=0; SKIP=flake8 poetry run pre-commit run --all-files
```

If we disable virtualenvs we don't need to specify poetry run
```suggestion
        run: export PIP_USER=0; SKIP=flake8 pre-commit run --all-files
```